cg/store/api/status.py

```diff
@@ -46,7 +46,7 @@ def samples_to_prepare(self) -> Query:
                 models.Application.is_external == False,
                 models.Sample.sequenced_at.is_(None),
             )
-            .order_by(models.Sample.priority.desc(), models.Sample.received_at)
```

Comment:
Priority is not important here anymore? Just double checking

Reply:
The view that relies on this code is not used much at all to my knowledge